packages/cli/src/polymer-cli.ts

```diff
@@ -132,6 +132,8 @@ export class PolymerCli {
         this.defaultConfigOptions.extraDependencies || [];
     this.defaultConfigOptions.extraDependencies.unshift(
         `bower_components${pathSeperator}webcomponentsjs${pathSeperator}*.js`);
+    this.defaultConfigOptions.extraDependencies.unshift(
+        `node_modules${pathSeperator}webcomponentsjs${pathSeperator}*.js`);
```

Comment:
@keanulee mentioned in chat that the bundles have moved to a separate folder, so this might need to be something like `node_modules${pathSeperator}webcomponentsjs${pathSeperator}**${pathSeparator}*.js`?

Reply:
Then again, I don't know what our glob syntax is. :/

Reply:
Oh, your right, this changes on npm